extension/snippets/xcomponent.json

```diff
@@ -0,0 +1,11 @@
+{
+	"Create a XComponent state": {
+		"prefix": "xcstate",
+		"description": "Create a XComponent state",
+    	"body": [
+            "<StateData Id=\"${1:index}\" Name=\"${2:state name}\" IsEntryPoint=\"${3|false,true|}\" SubGraphKey=\"${4:StateMachine + Id}\">",
```

Comment:
I guess you can remove the IsEntryPoint... (it is false by default)
and add it only on the component template.

Reply:
+1 I will do that in my next PR